runestone/activecode/js/MD5.js

```diff
@@ -0,0 +1,203 @@
+/**
```

Comment:
You just use the full http://..... form of the URL to link to it rather than a relative link.